src/Evolution/Systems/ScalarWave/BoundaryConditions/BoundaryCondition.cpp

```diff
@@ -0,0 +1,32 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/ScalarWave/BoundaryConditions/BoundaryCondition.hpp"
+
+#include <charm++.h>
```

Comment:
For my understanding, what is this included needed for?

Reply:
I had thought it was needed for `CkMigrateMessage` but this appears to be false, I'll remove :)